### PR TITLE
Update HDR tutorial

### PR DESCRIPTION
**Pull request by sobotka**
Fix some terminology and work towards a
more appropriate explanation of the process.

### CODE REVIEW

File: tutorials/3d/high_dynamic_range.rst [hunk 1]
```diff
@@ -11,89 +11,93 @@ looks at their awesome looking model in the 3D DCC and says "looks
...
+The mathematics of a scene referred model require that we multiply the scene by different
+values to adjust the intensities and exposure to different light ranges.
+The transfer function of the display cannot appropriately render
+the wider dynamic range of the game engine's scene output using the simple
+transfer function of the display however, and therefore a more complex approach
+to encoding is required.
 
-Linear color space & asset pipeline
------------------------------------
+Scene Linear & Asset Pipelines
```
Comment (cbscribe) [inline tutorials/3d/high_dynamic_range.rst]: Scene linear & asset pipelines

File: tutorials/3d/high_dynamic_range.rst [hunk 2]
```diff
@@ -11,89 +11,93 @@ looks at their awesome looking model in the 3D DCC and says "looks
...
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+There are two ways to do this:
 
-This is the most compatible way of using linear-space assets, and it will
-work everywhere, including all mobile devices. The main issue with this
-is loss of quality, as sRGB exists to avoid this same problem. Using 8
-bits per channel to represent linear colors is inefficient from the
-point of view of the human eye. These textures might later be compressed
-too, which makes the problem worse.
+sRGB Transfer Function to Display Linear Ratios on Image Import
```
Comment (cbscribe) [inline tutorials/3d/high_dynamic_range.rst]: Lower case

File: tutorials/3d/high_dynamic_range.rst [hunk 3]
```diff
@@ -11,89 +11,93 @@ looks at their awesome looking model in the 3D DCC and says "looks
...
 
-In any case, though, this is the easy solution that works everywhere.
+This is the easiest, but not the most ideal, method of using sRGB assets.
+One issue with this is loss of quality. Using 8 bit per channel to represent
+linear light ratios is not a sufficient but depth to quantise the values correctly.
+These textures might later be compressed too, which can exacerbate the problem.
 
-Hardware sRGB -> linear conversion
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Hardware sRGB Transfer Function to Display Linear Conversion
```
Comment (cbscribe) [inline tutorials/3d/high_dynamic_range.rst]: Lower case

File: tutorials/3d/high_dynamic_range.rst [hunk 4]
```diff
@@ -11,89 +11,93 @@ looks at their awesome looking model in the 3D DCC and says "looks
...
-This is the most correct way to use assets in linear-space, as the
-texture sampler on the GPU will do the conversion after reading the
+The GPU will do the conversion after reading the
 texel using floating point. This works fine on PC and consoles, but most
 mobile devices do no support it, or do not support it on compressed
 texture format (iOS for example).
 
-Linear -> sRGB at the end
-~~~~~~~~~~~~~~~~~~~~~~~~~
+Scene Linear to Display Referred Nonlinear
```
Comment (cbscribe) [inline tutorials/3d/high_dynamic_range.rst]: Lower case

**Comment by sobotka** [comment]
Updated.

**Comment by cbscribe** [comment]
Sorry for the delay in getting back to this. Thanks again for your work.

Also, in future, please squash your commits so that we have just one commit per PR. I'll go ahead and do it with this one.

**Pull request merged**